`ToggleButton` prop deprecation (#1701)
iTwin/stratakit: apps/website/src/content/docs/components/mui/ToggleButton.md

## StrataKit MUI modifications

Modifications to `ToggleButton`:

> - The `color`, `disableFocusRipple` and `fullWidth` props are deprecated and should not be used.

Author: ```suggestion
- The `color`, `disableFocusRipple` and `fullWidth` props are not supported.
```

- A `label` prop has been added. When specified, it is used as the **ToggleButton's** accessible name and is also shown in a tooltip on hover and focus.
- A `labelPlacement` prop has been added to control the placement of a tooltip that is shown when the `label` prop is specified.
- **ToggleButtons** are styled to match the visual appearance of the [**IconButton**](/components/iconbutton) component. Borders are displayed only when the buttons are wrapped in a `ToggleButtonGroup`. [Standalone](#standalone) **ToggleButtons** do not have borders.
- **ToggleButtons** can now be rendered as regular [**Buttons**](/components/button) to [display text](#text).

Modifications specific to `ToggleButtonGroup`:

> - The `color` prop is deprecated and should not be used.

Author: ```suggestion
- The `color` prop is not supported.
```
